src/iomarket.cpp

```diff
@@ -127,11 +127,10 @@ void processExpiredOffers(DBResult_ptr result, bool)
 				continue;
 			}
 
-			Player* player = g_game.getPlayerByGUID(playerId);
+			auto player = g_game.getPlayerByGUID(playerId);
```

Comment:
Could this be `const auto`
```suggestion
			const auto player = g_game.getPlayerByGUID(playerId);
```